README.md

```diff
@@ -50,7 +50,31 @@ Visit the [demo page](https://giraugh.github.io/hue-map/) to see a list with exa
 
 ### Custom colour maps
 
-> Coming soon...
+You can also provide a custom colour map to the `map` option with a type of `[number, number | RGBA][]`, an array of tuples, where each tuple has the index of where that colour appears in the gradient (from 0 to 1), and the colour at that point, as a HEX number or an RGBA tuple. Note that all HEX numbers need to include alpha.
```

Comment:
Might be a very long sentence perhaps? Might be able to be split into several but not a big deal.